.github/workflows/pr-build.yml

```diff
@@ -76,14 +83,17 @@ jobs:
         
       - name: Generate files with Builder
         run: dart run build_runner build --delete-conflicting-outputs 
+
+      - name: Dart static analysis
```

Comment:
Do you have any other names for this step?

Reply:
Uhm, not that I know of, the tool is called static check/analysis though

Reply:
Currently all other steps are in the form of "[verb] [object] [...]". To make it consistent, maybe do "Analyze Dart code" or something close?

Reply:
Ok English major, that works too